gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageNewIntegrationTest.java

```diff
@@ -692,7 +695,9 @@ public void listObjectInfo_includePrefix_objectInBucket() throws Exception {
   public void listObjectInfo_includePrefix_implicitDirInBucket() throws Exception {
     GoogleCloudStorage gcs = createGoogleCloudStorage(gcsOptions);
 
-    String testBucket = gcsfsIHelper.createUniqueBucket("lst-objs_incl-pfx_impl-dir-in-bckt");
+    String testBucket =
```

Comment:
why changing this?

Reply:
With bucket prefix changing from `gcsio-test` to `dataproc-gcs-gcsio` this was causing overall bucket name to go beyond limits.